sources/packages/backend/apps/db-migrations/src/migrations/1687553699796-EducationProgramsTableCreateOtherRegulatoryBody.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class EducationProgramsTableCreateOtherRegulatoryBody1687553699796
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Add-col-other-regulatory-body.sql", "EducationPrograms"),
```

Comment:
Just for the future, unless there is some limitation on the execution, I would not mind having the DB changes altogether in one SQL script that can be applied/rollback altogether.